packages/components/pages/f-registration/package.json

```diff
@@ -51,14 +51,14 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.4.0",
+    "@justeat/f-services": "1.x",
     "vuelidate": "0.7.6"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "@justeat/f-button": "3.x",
+    "@justeat/f-button": "4.x",
```

Comment:
This should've been updated to `4.x` a while back. As you can see on line 68, we already test f-registration with `f-button@4.0.0`, so we know it's compatible with this version 👍🏼